sycl/doc/extensions/experimental/sycl_ext_oneapi_enqueue_functions.asciidoc

```diff
@@ -709,6 +709,32 @@ optimize such partial barriers.
 _{endnote}_]
 |====
 
+==== Command Graph
+
+The functions in this section are only available if the
+link:./sycl_ext_oneapi_graph.asciidoc[
+  sycl_ext_oneapi_graph] extension is supported.
+
+|====
+a|
+[frame=all,grid=none]
+!====
+a!
+[source,c++]
+----
+namespace sycl::ext::oneapi::experimental {
+
+template <typename CommandGroupFunc>
+void submit(command_graph<graph_state::modifiable> g, CommandGroupFunc&& cgf);
```

Comment:
Is this supposed to correspond to `command_graph::add`?  That's the API you use to build a graph in explicit mode.  If that is the case, then the name of this API seems wrong.  No commands are actually being submitted with this API.

Reply:
Yes this maps to `add`. Naming is a compromise again between seamless integration into enqueue function extension and its current SYCL-like naming scheme: `submit*`. 

Another option could be something like `add_in_order` or `add_linear`, indicating the implicit creation of edges.

Reply:
Why does this API implicitly create a linear graph?  The corresponding function `submit(sycl::queue q)` doesn't imply a linear set of commands.  Why should it be different when building a graph?

Regarding naming, how bout `add_graph_node` (assuming it does not have the linear semantic)?